ConsumerAffairs/django-urlographer
# using site name instead of domain in choice field and dropping django 1.5 support

## Test
- Go to /admin/urlographer/urlmap/add/
- Make sure the Site choice field shows the site name and domain instead of just the domain
